src/components/index.js

```diff
@@ -12,38 +12,42 @@ export {
     PlusIcon,
     TasksIcon,
     TEIIcon,
-} from './Icon/Icon'
-export { DataElementIdScheme } from './ElementSchemes/DataElementIdScheme'
-export { DatePicker } from './DatePicker/DatePicker'
-export { DatePickerField } from './DatePicker/DatePickerField'
-export { EventIdScheme } from './ElementSchemes/EventIdScheme'
-export { FileUpload } from './FileUpload/FileUpload'
-export { FormAlerts } from './FormAlerts/FormAlerts'
-export { FormField } from './FormField/FormField'
-export { IdScheme } from './ElementSchemes/IdScheme'
-export { ImportButtonStrip } from './ImportButtonStrip/ImportButtonStrip'
-export { JobOverview } from './JobOverview/JobOverview'
-export { JobSummary } from './JobSummary/JobSummary'
-export { MiniJobSummary } from './JobSummary/MiniJobSummary/MiniJobSummary'
-export { MoreOptions } from './MoreOptions/MoreOptions'
-export { OrgUnitIdScheme } from './ElementSchemes/OrgUnitIdScheme'
-export { OrgUnitTreeField } from './OrgUnitTree/OrgUnitTreeField'
-export { Page } from './Page/Page'
-export { ProgramStages } from './ProgramStages/ProgramStages'
-export { RadioGroup } from './RadioGroup/RadioGroup'
-export { RadioGroupField } from './RadioGroup/RadioGroupField'
-export { Router } from './Router/Router'
-export { Schemas } from './Schemas/Schemas'
-export { SchemasField } from './Schemas/SchemasField'
-export { Select } from './Select/Select'
-export { SelectField } from './Select/SelectField'
-export { Sidebar } from './Sidebar/Sidebar'
-export { Switch } from './Switch/Switch'
-export { WithAuthority } from './WithAuthority/WithAuthority'
-export { DurationField } from './Duration/DurationField'
-export { ResourcePicker } from './ResourcePicker/ResourcePicker'
-export { ResourcePickerField } from './ResourcePicker/ResourcePickerField'
-export { BasicOptions } from './BasicOptions/BasicOptions'
-export { SchemeContainer } from './ElementSchemes/SchemeContainer'
-export { StyledField } from './StyledField/StyledField'
-export { ValidationSummary } from './ValidationSummary/ValidationSummary'
+} from './Icon/Icon.js'
```

Comment:
please edit this cherry-pick manually (to remove `.js` and minimise the change log) - this is an example of where backports become annoying for older codebases.

Reply:
Okay, I can do that. I thought about not making that change but the linter had complained about the lack of the `.js` extension. Let me fix that and update